recipes/nextalign/meta.yaml

```diff
@@ -6,11 +6,13 @@ package:
   version: "{{ version }}"
 
 source:
-  url: https://github.com/nextstrain/nextclade/releases/download/nextalign-{{ version }}/nextalign-Linux-x86_64
-  sha256: ea683139194d4041d52edc92a9653dcf77fa6c106281db0031bde6025a479f37
+  - url: https://github.com/nextstrain/nextclade/releases/download/nextalign-{{ version }}/nextalign-Linux-x86_64 # [linux64]
+    sha256: ea683139194d4041d52edc92a9653dcf77fa6c106281db0031bde6025a479f37                                      # [linux64]
+  - url: https://github.com/nextstrain/nextclade/releases/download/nextalign-{{ version }}/nextalign-MacOS-x86_64 # [osx]
+    sha256: f02adff831578e5297c5a8e05dbc682c335a31e6bd6c21077493aa3c1fa33ca9                                      # [osx]
 
 build:
-  number: 0
+  number: 1
   skip: True # [(not (linux and x86_64)) or osx]
```

Comment:
This needs amending to no longer skip osx

Reply:
Thank you for catching this, @cjw85! I just pushed a fix for this and added support for the arm64 platform.